de_workloads/ingest/Ingest_AzureSql_Example/data_factory/pipelines/Ingest_AzureSql_Example.json

```diff
@@ -0,0 +1,237 @@
+                                                    "type": "Expression"
+                                                },
+                                                "filename": {
+                                                    "value": "@{item()['display_name']}.parquet",
+                                                    "type": "Expression"
+                                                }
+                                            }
+                                        }
+                                    ]
+                                },
+                                {
+                                    "name": "Generate_Ingest_Query",
+                                    "type": "ExecutePipeline",
+                                    "dependsOn": [],
+                                    "userProperties": [],
+                                    "typeProperties": {
+                                        "pipeline": {
+                                            "referenceName": "Generate_Ingest_Query",
+                                            "type": "PipelineReference"
+                                        },
+                                        "waitOnCompletion": true,
+                                        "parameters": {
+                                            "ingest_entity_config": {
+                                                "value": "@item()",
+                                                "type": "Expression"
+                                            },
+                                            "window_start": {
+                                                "value": "@pipeline().parameters.window_start",
+                                                "type": "Expression"
+                                            },
+                                            "window_end": {
+                                                "value": "@pipeline().parameters.window_end",
+                                                "type": "Expression"
+                                            }
+                                        }
+                                    }
+                                }
+                            ]
+                        }
+                    }
+                ],
+                "policy": {
+                    "elapsedTimeMetric": {},
+                    "cancelAfter": {}
+                },
+                "parameters": {
+                    "data_source_name": {
+                        "type": "string",
+                        "defaultValue": "Ingest_AzureSql_Example"
+                    },
+                    "window_end": {
+                        "type": "string",
+                        "defaultValue": "2020-01-01"
+                    },
+                    "window_start": {
+                        "type": "string",
+                        "defaultValue": "2020-01-31"
+                    },
```

Comment:
Is it correct that window_start > window_end?

Reply:
No good spot - will update however this links to the issue I raised that the E2E tests are not reliable currently (ticket to be added).